pkg/pillar/devicenetwork/devicenetwork.go

```diff
@@ -185,9 +193,10 @@ func MakeDeviceNetworkStatus(globalConfig types.DevicePortConfig, oldStatus type
 		globalStatus.Ports[ix].DomainName = u.DomainName
 		globalStatus.Ports[ix].NtpServer = u.NtpServer
 		globalStatus.Ports[ix].DnsServers = u.DnsServers
-		if u.ParseError != "" {
-			globalStatus.Ports[ix].Error = u.ParseError
-			globalStatus.Ports[ix].ErrorTime = u.ParseErrorTime
+		if u.Error != "" {
+			globalStatus.Ports[ix].Error = u.Error
+			globalStatus.Ports[ix].ErrorTime = u.ErrorTime
+			continue
```

Comment:
No, because if you have IP addresses etc you want to report those and that is further down in this function.

Reply:
Hmm.. Probably a conflict resolution error.. Not sure why I added the continue here.. Will fix it..

Reply:
Fixed

Reply:
@kalyan-nidumolu this comment is 15 days old and needs to be addressed!